test/addFreakTest.js

```diff
@@ -0,0 +1,88 @@
+const { expect, assert } = require("chai");
+const EventEmitter = require('events');
+
+describe("FreakTeam", function () {
```

Comment:
Please use a stated style guide for javascript code. Also, use a hinter for enforcing the style.